package_template.json

```diff
@@ -15,7 +15,7 @@
   },
   "devDependencies": {
     "sapper": "^0.28.0",
-    "svelte": "^3.17.3"
+    "svelte": "^3.30.0"
```

Comment:
This shouldn't be necessary, and I'm against making this change, as it could mask unintended peerdep changes. 3.17.3 was selected because it was necessary for making deduping `<head>` elements during hydration the responsibility of Svelte rather than Sapper. But there's nothing in the upgrade of rollup-plugin-svelte that ought to require a version of Svelte newer than 3.5.0, which it already was.

Reply:
Ok, I changed it back. I figured they'll get 3.30 either way since there's no `package-lock.json` and it just made it look a little more up-to-date. It shouldn't have an effect one way or the other and you'll probably get the masking issue either way